xmrstak/cli/cli-miner.cpp

```diff
@@ -299,7 +299,7 @@ int main(int argc, char *argv[])
 		}
 		if(opName.compare("-v") == 0 || opName.compare("--version") == 0)
 		{
-			std::cout<< "Version: " << get_version_str() << std::endl;
+			std::cout<< "Version: " << get_version_str_short() << std::endl;
```

Comment:
please keep the long version that we have all information if someone is opening an issue.

Reply:
If you will keep it in short form we need to add a flag for the long version. This is really important to reduce the time to solve an issue.